readme.md

```diff
@@ -89,6 +89,7 @@ $ cpy --help
 
   Example
     $ cpy 'src/*.png' dist
+    $ cpy 'src/*.png' '!src/*.ico' dist
```

Comment:
@sindresorhus updated today the [readme.md](https://github.com/sindresorhus/cpy/commit/800b0091a9932ad3bad46b4ea5e188402d5e0433), so I would suggest:
`$ cpy 'src/*.png', '!src/goat.png' dist`